src/language/branches.md

```diff
@@ -113,7 +115,8 @@ switch (command) {
     executeNowClosed();   // Runs for both OPEN and PENDING.
 }
 ```
-For an empty case that does not fall through, use [`break`][break] for its body.
+
+For more details, check out the patterns [section on switch][].
```

Comment:
It might be helpful to add more information on what kind of details, otherwise I am not sure why I would click this link. Depending on what you are thinking, something like:

```suggestion
To learn more about patterns in case clauses, 
check out the patterns documentation on [Switch statements and expressions][].
```